bower.json

```diff
@@ -34,5 +35,8 @@
         "sinon": "http://sinonjs.org/releases/sinon-1.17.0.js",
         "angular-mocks": "1.4.4"
     },
-    "private": true
+    "private": true,
+    "resolutions": {
```

Comment:
<img width="676" alt="screen shot 2015-11-20 at 10 35 05 am" src="https://cloud.githubusercontent.com/assets/918514/11303729/63dca96e-8f72-11e5-80d0-a78973354839.png">
it seems like the only issue could be bootstrap-timepicker since it is purporting that it needs jquery 2.0. everything else seems to be fine using `1.11.1`. if you've found that the timepicker works as you need it to with `1.11.1` i'd say it's fine to leave it as is. it'd be way harder to upgrade jquery to 2.0 at the moment.

alternatively if you find you do need jquery 2.0, you can install a new lib called jquery-latest and then make sure that's loaded on the page when bootstrap-timepicker is loaded. (you'd also need to ensure that doesn't break any other existing functionality)